azure/resource_groups/app/parameters/development.template.json

```diff
@@ -139,6 +139,14 @@
         "secretName": "DfeSignInApiEndpoint"
       }
     },
+    "GOOGLE_ANALYTICS_ID": {
+      "reference": {
+        "keyVault": {
+          "id": "${keyVaultId}"
+        },
+        "secretName": "GoogleAnalyticsId"
```

Comment:
We could argue that this doesn't need to be stored in KeyVault as it's not secret. We could just provide it as a value?

Reply:
Putting it in the Key Vault gives us some advantages in that we can change it and redeploy without committing changes. Though it does add some indirection, and increase the number of "secrets" we store that aren't secret.